src/player.js

```diff
@@ -1,35 +1,19 @@
 import { GameBoard } from "./gameBoard";
```

Comment:
This file change does not need to be reviewed.  Essentially, my player object is much more simplified after my new computer which is why there are more deletions than insertions